setup.py

```diff
@@ -6,38 +6,8 @@
 ################################################################################
 
 import setuptools
-import os
 import sys
 import warnings
-import skforecast
-
-VERSION = skforecast.__version__
-
-with open('requirements.txt') as f:
-    requirements_base = f.read().splitlines()
-
-with open('requirements_optional.txt') as f:
-    requirements_optional = f.read()
-
-with open("requirements_test.txt") as f:
-    requirements_test = f.read().splitlines()
-
-extras_require = {
-    "sarimax": requirements_optional.split("\n\n")[0].splitlines(),
-    "plotting": requirements_optional.split("\n\n")[1].splitlines(),
-    "test": requirements_test
-}
-
-extras_require["full"] = (
-    extras_require["sarimax"]
-    + extras_require["plotting"]
-    + extras_require["test"]
-)
-
-extras_require["all"] = extras_require["full"]
-
-def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 if sys.version_info[:2] < (3, 8):
```

Comment:
With `requires_python` in `pyproject.toml`, this part is redundant and probably `setup.py` can be removed completely 😄